test-it/testList.json

```diff
@@ -361,7 +361,13 @@
                 "TestVI_SimpleArrayOfRefs.via",
                 "TestVI_SpreadSheetStringToArray.via",
                 "TestVI_UnflattenFromJSONInvalidCluster.via",
```

Comment:
TestVI_CompatibleDataTypes_VariantToData_DoesNotReportError.via
is not in testlist.json

Reply:
Accidentally added it. Removing the files.